version/command.go

```diff
@@ -1,31 +1,48 @@
 package version
 
 import (
+	"encoding/json"
 	"fmt"
-	"runtime"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/tendermint/tendermint/libs/cli"
+)
+
+const (
+	flagLong = "long"
 )
 
 var (
+
 	// VersionCmd prints out the current sdk version
 	VersionCmd = &cobra.Command{
 		Use:   "version",
 		Short: "Print the app version",
-		Run:   printVersion,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			verInfo := newVersionInfo()
+
+			if !viper.GetBool(flagLong) {
+				fmt.Println(verInfo.CosmosSDK)
+				return nil
+			}
+
+			if viper.GetString(cli.OutputFlag) != "json" {
```

Comment:
Hmm what if we add another output type in the future?

Reply:
I think that would require a significant refactor beyond what we have now.

Reply:
Many things will need to be amended then